docs/development/testing.md

```diff
@@ -76,7 +76,7 @@ To run common benchmarks to understand Pyodide's performance, begin by
 installing the same prerequisites as for testing. Then run:
 
 ```bash
-make benchmark
+PYODIDE_PACKAGES="numpy,matplotlib" make benchmark
```

Comment:
`make benchmark` wasn't properly working after we've updated to the minimal build by default (https://github.com/pyodide/pyodide/pull/1804)